### PR TITLE
Define cirq.ISWAP_INV

### LINKED ISSUE #4615: Define `cirq.ISWAP_INV`, similar to `cirq.SQRT_ISWAP_INV` and replace usages.
**Description of the issue**
Similar to `cirq.SQRT_ISWAP_INV`, define a global constant for `cirq.ISWAP_INV` and replace usages of `cirq.ISWAP ** -1` with `cirq.ISWAP_INV`. 

https://github.com/quantumlib/Cirq/blob/7951f84a0110be719c8a9e07aa1ffec2b0a31990/cirq-core/cirq/ops/swap_gates.py#L356

**Cirq version**
0.14.0

### PR DESCRIPTION
**Pull request by unaiic**
Define a global constant for `cirq.ISWAP_INV`, similar to `cirq.SQRT_ISWAP_INV`, and replace usages of `cirq.ISWAP ** -1` with `cirq.ISWAP_INV`.

Fixes #4615.

### CODE REVIEW

File: cirq-google/cirq_google/serialization/common_serializers_test.py [hunk 1]
```diff
@@ -551,7 +551,7 @@ def test_wait_gate_multi_qubit():
         (cirq.ISWAP ** 0.5, -np.pi / 4, 0),
         (cirq.ISWAP ** -0.5, np.pi / 4, 0),
         (cirq.ISWAP ** 1.0, -np.pi / 2, 0),
-        (cirq.ISWAP ** -1.0, np.pi / 2, 0),
+        (cirq.ISWAP_INV, np.pi / 2, 0),
```
Comment (dabacon) [inline cirq-google/cirq_google/serialization/common_serializers_test.py:554]: nit: move down below other ISWAP.   Also considering adding another case like cirq.ISWAP_INV ** 0.5

**Review by dabacon — APPROVED** [review]

**Review by tanujkhattar — CHANGES_REQUESTED** [review]
Please create `cirq-core/cirq/protocols/json_test_data/ISWAP_INV.json` and `cirq-core/cirq/protocols/json_test_data/ISWAP_INV.repr` to fix the tests. You can look at existing json and repr files for `ISWAP` for reference.  

Please also add tests to cirq-core/cirq/ops/swap_gates_test.py by extending the existing tests for `cirq.SWAP`.

**Comment by tanujkhattar** [comment]
@unaiic Gentle reminder. Please address the comments and fix the failing tests so that we can merge the PR.

**Comment by dabacon** [comment]
Implemented in #5613

**Pull request closed without merging**